var/spack/repos/builtin/packages/archer/package.py

```diff
@@ -35,3 +37,33 @@ def cmake_args(self):
             '-DCMAKE_CXX_COMPILER=clang++',
             '-DOMP_PREFIX:PATH=%s' % self.spec['llvm-openmp-ompt'].prefix,
         ]
+
+    @run_after('install')
+    def cache_test_sources(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(['test'])
+
+    def run_parallel_example_test(self):
+        """Run stand alone test: parallel-simple"""
+
+        test_dir = join_path(self.install_test_root, 'test', 'parallel')
```

Comment:
Please use the test stage copy for stand-alone tests:

```suggestion
        test_dir = join_path(self.test_suite.current_test_cache_dir, 'test', 'parallel')
```